RELEASING.md

```diff
@@ -32,23 +32,15 @@ tools must be installed.
 Release Steps
 -------------
 
-### Preparing Documentation
+### Release Documentation update
 
-Update version-sensitive files in the root and documentation sources
-in the `docs` subdirectory:
-
-* `docs/changelog.txt`
-* `docs/versions.txt`
-* `docs/upgrade.txt`
-
-Use the [`docs/build-and-copy-docs.sh`](docs/build-and-copy-docs.sh) script to
-build a set of docs for this release and copy them to the cloned
-`docs.janusgraph.org` repo which you will update later.
-
-You may also need to update the following files in the main repo for any new
-or updated dependencies:
-
-* `NOTICE.txt`
+1. Update version-sensitive files in the root and documentation sources in the `docs` subdirectory: 
```

Comment:
(nitpick) It's just a single file now so it doesn't make sense any more to write `files` and then use an enumeration.